app/containers/NewRequestPage.js

```diff
@@ -13,7 +13,12 @@ class NewRequestPage extends Component {
   }
 
   componentDidMount() {
-    this.setState({request: new Request({method: 'GET'})});
+    const newRequest = new Request({
+      method: 'GET',
+      projectId: this.props.params.projectId
```

Comment:
as mentioned in the PR description we do need a proper way to distinguish between historical and non-historical requests. Played a bit with it but not too happy. Lets pair on this if u have the chance